speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
Then that is strange, maybe it is a difference when using python-Levenshtein and fuzzy matching. Though it shouldn't depend on it.

Using the articles in the sample utterances  

![screen shot macbook pro 2017-01-11 at 18 35 35](https://cloud.githubusercontent.com/assets/5680968/21861595/6e98ac36-d82d-11e6-876d-3787d653a199.png)

Works as expected

![screen shot macbook pro 2017-01-11 at 18 41 01](https://cloud.githubusercontent.com/assets/5680968/21861623/8aed8974-d82d-11e6-9dd1-13af872f79d1.png)